tests-functional/tests/test_activity_center_notifications.py

```diff
@@ -0,0 +1,190 @@
+import pytest
+
+from tests.test_cases import MessengerTestCase
+
+
+@pytest.mark.usefixtures("setup_two_unprivileged_nodes")
+@pytest.mark.rpc
+class TestActivityCenterNotifications(MessengerTestCase):
+
+    def test_activity_center_notifications(self):
+        message_id = self.send_contact_request_and_wait_for_signal_to_be_received()
+        response = self.receiver.wakuext_service.rpc_request(
+            method="activityCenterNotifications", params=[{"cursor": "", "limit": 20, "activityTypes": [5], "readType": 2}]
+        ).json()
+        self.receiver.verify_json_schema(response, method="wakuext_activityCenterNotifications")
+        result = response["result"]
+        assert all(
+            (
+                result["notifications"][0]["accepted"] is False,
+                result["notifications"][0]["author"] == self.sender.public_key,
+                result["notifications"][0]["chatId"] == self.sender.public_key,
+                result["notifications"][0]["id"] == message_id,
+                result["notifications"][0]["read"] is False,
+                result["notifications"][0]["lastMessage"]["contactRequestState"] == 1,
+            )
+        )
+
+        self.accept_contact_request_and_wait_for_signal_to_be_received(message_id)
+        response = self.sender.wakuext_service.rpc_request(
+            method="activityCenterNotifications", params=[{"cursor": "", "limit": 20, "activityTypes": [1, 2, 3, 4, 5, 7, 8, 9, 10, 23, 24]}]
+        ).json()
+        self.sender.verify_json_schema(response, method="wakuext_activityCenterNotifications")
+        result = response["result"]
+        assert all(
+            (
+                result["notifications"][0]["accepted"] is True,
+                result["notifications"][0]["author"] == self.sender.public_key,
+                result["notifications"][0]["chatId"] == self.receiver.public_key,
+                result["notifications"][0]["id"] == message_id,
+                result["notifications"][0]["read"] is True,
+                result["notifications"][0]["message"]["contactRequestState"] == 2,
+                result["notifications"][0]["lastMessage"]["text"] == f"@{self.receiver.public_key} accepted your contact request",
```

Comment:
more like personal preference, maybe we can add to variable notification = response["result"]["notifications"][0] to lighten up it a bit